packages/edit-post/src/components/manage-blocks-modal/style.scss

```diff
@@ -75,6 +75,10 @@
 		font-size: 0.9rem;
 		font-weight: 600;
 	}
+
+	> a {
+		float: right; //Aligning link to the right of category heading.
```

Comment:
It looks like using `float` changes only the visual appearance but not the tabbing order:
![tab-order-block-manager](https://user-images.githubusercontent.com/699132/71894089-9b8a5b00-314d-11ea-9477-eed10634cefd.gif)

We need to find a different approach here.

Reply:
Yeah, I was wondering about that. `float:right` works easier this way, but tab order is off now. I'll fix this.